test/verify/check-openshift

```diff
@@ -652,7 +652,7 @@ class TestRegistry(MachineCase):
             b.click(".btn-primary")
             b.wait_not_present("modal-dialog")
             b.wait_present("table.listing-ct")
-            wait(lambda: username not in o.execute("oc get rolebinding -n testprojectuserproj"))
+            wait(lambda: username not in o.execute("oc get rolebinding -n testprojectuserproj"), delay=5)
```

Comment:
It seems a bit odd that this would regulary take more than one minute.. I'm fine with increasing the delay of course if that helps, but I'd actually expect this to take much less than a second.